src/main/java/org/opensearch/security/rest/SecurityHealthAction.java

```diff
@@ -65,7 +66,7 @@ public SecurityHealthAction(final Settings settings, final RestController contro
 
     @Override
     public List<Route> routes() {
-        return routes;
+        return Utils.addRoutesPrefix(routes, "/_opendistro/_security", "/_plugins/_security");
```

Comment:
?

Reply:
The APIs in this folder, does nor have `/api/` in their routes, so we have to provide customized paths.

Reply:
My question is why it applies to `routes()` and not `routes`?